docs/assistant/connect-to-openai.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[assistant-connect-to-openai]]
+= Connect to OpenAI
+
+This page provides step-by-step instructions for setting up an OpenAI connector for the first time. This connector type enables you to leverage OpenAI's large language models (LLMs) within {kib}. You'll first need to create an OpenAI API key, then configure the connector in {kib}. 
+
+
+[discrete]
+== Configure OpenAI
+
+[discrete]
+=== Select a model
+
+Before creating an API key, you must choose a model. Refer to the https://platform.openai.com/docs/models/gpt-4-turbo-and-gpt-4[OpenAI docs] to select a model. Take note of the specific model name (for example `gpt-4-turbo`); you'll need it when configuring {kib}.
+
+NOTE: `GPT-4 Turbo` offers increased performance. `GPT-4` and `GPT-3.5` are also supported. 
+
+[discrete]
+=== Create an API key
+
+To generate an API key:
+
+. Log in to the OpenAI platform and navigate to **API keys**.
+. Select **Create new secret key**.
+. Name your key, select an OpenAI project, and set the desired permissions.
+. Click **Create secret key** and then copy and securely store the key. It will not be accessible after you leave this screen.
+
+The following video demonstrates these steps.
+
+=======
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js"></script>
+<img
+  style="width: 100%; margin: auto; display: block;"
+  class="vidyard-player-embed"
+  src="https://play.vidyard.com/vbD7fGBGgyxK4TRbipeacL.jpg"
+  data-uuid="vbD7fGBGgyxK4TRbipeacL"
+  data-v="4"
+  data-type="inline"
+/>
+</br>
+++++
+=======
+
+[discrete]
+== Configure the OpenAI connector
+
+To integrate with {kib}:
+
+. Log in to {kib}.
+. Navigate to **Stack Management → Connectors → Create Connector → OpenAI**.
+. Provide a name for your connector, such as `OpenAI (GPT-4 Turbo Preview)`, to help keep track of the model and version you are using.
+. Under **Select an OpenAI provider**, choose **OpenAI**.
+. The **URL** field can be left as default.
```

Comment:
Add this line after . The **URL** field can be left as default.
In the default model specify your [model](https://platform.openai.com/docs/models/gpt-4-turbo-and-gpt-4) of choice. This section can be updated later on to reflect new releases in models.